network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -58,16 +58,18 @@ <h1 class="tw-h1-heading {% if uses_menu %}tw-hidden medium:tw-block{% endif %}"
 
                     {% block cta %}
                         {% if page.cta != None %}
-                            <div class="cta tw-pb-4
+                            <div class="cta tw-pb-4 tw-mx-auto
```

Comment:
This `tw-mx-auto` class keeps the CTA form centered on mobile and tablet breakpoints